Rakefile

```diff
@@ -1,54 +1,88 @@
-require 'voxpupuli/test/rake'
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.